backend/src/services/service-token/service-token-service.ts

```diff
@@ -200,15 +200,15 @@ export const serviceTokenServiceFactory = ({
         try {
           await smtpService.sendMail({
             recipients: [token.createdByEmail],
-            subjectLine: "Service Token Expired",
+            subjectLine: "Your Service Token is about to expire",
```

Comment:
the casing is a bit off here

also, thoughts on "Service Token Expiry Notice" instead?